kitchen/modules/adding-machine/src/lib.rs

```diff
@@ -32,3 +32,128 @@ decl_event!(
         Added(u32, u32, u32),
     }
 );
+
+#[cfg(test)]
+mod tests {
+    use crate::{Module, Trait};
+    use primitives::H256;
+    use runtime_io;
+    use runtime_primitives::{
+        testing::Header,
+        traits::{BlakeTwo256, IdentityLookup},
+        Perbill,
+    };
+    use support::{assert_err, impl_outer_event, impl_outer_origin, parameter_types, traits::Get};
+    use system::{EventRecord, Phase};
+
+    impl_outer_origin! {
+        pub enum Origin for Runtime {}
+    }
+
+    // Workaround for https://github.com/rust-lang/rust/issues/26925 . Remove when sorted.
+    #[derive(Clone, PartialEq, Eq, Debug)]
+    pub struct Runtime;
+    parameter_types! {
+        pub const BlockHashCount: u64 = 250;
+        pub const MaximumBlockWeight: u32 = 1024;
+        pub const MaximumBlockLength: u32 = 2 * 1024;
+        pub const AvailableBlockRatio: Perbill = Perbill::one();
+    }
+    impl system::Trait for Runtime {
+        type Origin = Origin;
+        type Index = u64;
+        type Call = ();
+        type BlockNumber = u64;
+        type Hash = H256;
+        type Hashing = BlakeTwo256;
+        type AccountId = u64;
+        type Lookup = IdentityLookup<Self::AccountId>;
+        type Header = Header;
+        type Event = TestEvent;
+        type BlockHashCount = BlockHashCount;
+        type MaximumBlockWeight = MaximumBlockWeight;
+        type MaximumBlockLength = MaximumBlockLength;
+        type AvailableBlockRatio = AvailableBlockRatio;
+        type Version = ();
+    }
+
+    mod added {
+        pub use crate::Event;
```

Comment:
could also be

```suggestion
        pub use super::super::*;
```

what is preferred?

Reply:
I vote for `pub use crate::Event;`

Reply:
cool, I agree. Just changed all `super::super::*` to `crate::Event`.